packages/flutter/lib/src/widgets/locale_query.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'framework.dart';
 
-// Superclass for locale-specific data provided by the application.
+/// Superclass for locale-specific data provided by the application.
 class LocaleQueryData { }
 
 class LocaleQuery<T extends LocaleQueryData> extends InheritedWidget {
```

Comment:
We really should consider renaming this class to something generic. It has nothing Locale-specific IIRC.